### PR TITLE
Replaces axios with fetch

### PR DESCRIPTION
**Pull request by rschwabco**

### CODE REVIEW

File: tests/index.test.ts [hunk 1]
```diff
@@ -175,38 +178,58 @@ describe('Pinecone Client Index Operations', () => {
...
-    await index.delete1([], true, namespace)
-    const fetchResult = await index.fetch([...vectors.map((v) => v.id)], namespace)
-    expect(Object.keys(fetchResult.data.vectors as object).length).toBe(0)
+    await index.delete1({
+      deleteAll: true,
+      namespace
+    })
+    const fetchResult = await index.fetch({
+      ids: [...vectors.map((v) => v.id)]
+      , namespace
```
Comment (damargulis) [inline tests/index.test.ts:225]: super nit. fix the spacing here

**Review by damargulis — APPROVED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -1,13 +1,11 @@
 import axios from 'axios'
```
Comment (fraserxu) [inline src/index.ts:1]: will this get replaced in a separate pull request?

**Review by zaeemadamjee — APPROVED** [review]

File: src/index.ts (hunk 2 above)
Comment (rschwabco) [reply src/index.ts]: And it's gone. Sorry about that.

**Comment by fraserxu** [comment]
💯

**Pull request merged**